vendor-extra/LiberoPatternsBundle/src/Resources/views/site-header.html.twig

```diff
@@ -0,0 +1,23 @@
+{%- import '@LiberoPatterns/html.html.twig' as html -%}
+
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['site-header']) }) -%}
+
+<header {{- html.attributes(attributes) }}>
+
+    {%- set logo = logo|default({}) -%}
+    {%- with logo -%}
+
+        {%- set href = logo.href|default('') -%}
+        {%- set image = logo.image|default({})|merge({src: asset(logo.image.src|default('images/libero-logo.svg')), alt: logo.image.alt|default('Site logo')}) -%}
```

Comment:
`asset()` should only be used on pattern asset paths (ie `images/libero-logo.svg`). There’s a rewrite that hasn’t happened as a result, so this won’t work as is.